test/test_loop_rotation.cpp

```diff
@@ -605,39 +605,39 @@ __global__ void CUDAGeneratedKernel(Tensor<float, 2> T0, Tensor<float, 2> T3) {
      = T4[0];
   #pragma unroll 1
   for(nvfuser_index_t i19 = 0; i19 < T0.size[0]; ++i19) {
-    float* ptr383;
-    ptr383 = ptr382 + (T0.stride[0] * i19);
-    int64_t i568;
-    i568 = 4 + i19;
-    unsigned i571;
-    i571 = (toSmem(T4)) + (12 * (i568 % 5));
-    int64_t i657;
-    i657 = 1 + (3 * (i19 % 5));
-    int64_t i1029;
-    i1029 = 3 * i19;
-    bool b1517;
-    b1517 = i568 < T0.size[0];
+    float* ptr2292;
+    ptr2292 = ptr2237 + (T0.stride[0] * i19);
+    int64_t i2765;
+    i2765 = 4 + i19;
+    unsigned i2768;
+    i2768 = (toSmem(T4)) + (12 * (i2765 % 5));
+    int64_t i3667;
+    i3667 = 1 + (3 * (i19 % 5));
+    int64_t i5370;
+    i5370 = 3 * i19;
+    bool b7647;
+    b7647 = i2765 < T0.size[0];
     #pragma unroll
     for(nvfuser_index_t i17 = 0; i17 < 3; ++i17) {
-      Ampere::cpAsyncCa<float, 1>((i571 + (4 * i17)),(ptr383 + (T0.stride[1] * (i17 + nvfuser_zero))),b1517);
+      Ampere::cpAsyncCa<float, 1>((i2768 + (4 * i17)),(ptr2292 + (T0.stride[1] * (i17 + nvfuser_zero))),b7647);
     }
     NVFUSER_UPDATE_MAGIC_ZERO
     Ampere::cpAsyncCommit();
     #pragma unroll
     for(nvfuser_index_t i22 = 0; i22 < 2; ++i22) {
       T1[((1 + i22) % 2)]
-         = T4[(i657 + i22)];
+         = T4[(i3667 + i22)];
       float T2[1];
       T2[0]
          = T1[(i22 % 2)];
-      T3[(i1029 + (i22 + nvfuser_zero))]
+      T3[(i5370 + (i22 + nvfuser_zero))]
```

Comment:
Wow, `5370`! Hopefully, it won't overflow 😄

Reply:
Maybe at some point we would need to implement some garbage collection mechanism to reduce memory usage...